integration-test/utils/datafactory/footer.data.ts

```diff
@@ -0,0 +1,35 @@
+export const footerExpectedInformation = {
+    "title": "Follow Us",
+    "subtitle": "Join us on social media and stay tuned.",
+    "description": "Women Coding Community is a not-for-profit organisation. © 2024 Women Coding Community",
+    "network": [{
+        "type": "linkedIn",
+        "link": "https://www.linkedin.com/company/womencodingcommunity"
```

Comment:
I think that should be up you QA decided, what I am trying to avoid is that in case we change the value of this attribute, it should not break the API contract in my opinion. Because the API is flexible to change the values, in my opinion we should validate if it is a valid link and if it is present. Because, let's image in case we decided for some reason rename the linkedIn account it should be ok.